pkg/utils/sectmpl/sectmpl.go

```diff
@@ -0,0 +1,168 @@
+/*
```

Comment:
just rename to `pkg/utils/template/secrets.go`

Reply:
Should be renamed